src/infile_reader.cc

```diff
@@ -143,17 +143,34 @@ namespace format {
         boost::algorithm::trim(str);
         conf.output_score_threshold = boost::lexical_cast<fltype>(str);
       }
-      else if (boost::algorithm::starts_with(buffer, "POSE_RMSD ")) {
-        std::string str = buffer.substr(10);
+      else if (boost::algorithm::starts_with(buffer, "MIN_RMSD ")) {
+        std::string str = buffer.substr(9);
         boost::algorithm::trim(str);
-        conf.pose_rmsd = boost::lexical_cast<fltype>(str);
+        conf.pose_min_rmsd = boost::lexical_cast<fltype>(str);
       }
       else if (boost::algorithm::starts_with(buffer, "NO_LOCAL_OPT ")) {
         std::string str = buffer.substr(13);
         boost::algorithm::trim(str);
         std::transform(str.begin(), str.end(), str.begin(), ::tolower);
         conf.no_local_opt = (str != "false");
       }
+      else if (boost::algorithm::starts_with(buffer, "SCORE_ONLY ")) {
+        std::string str = buffer.substr(11);
+        boost::algorithm::trim(str);
+        std::transform(str.begin(), str.end(), str.begin(), ::tolower);
+        conf.score_only = (str != "false");
```

Comment:
この条件ですが、うーん、どうするべきかな…。タイプミスをすると強制的に score_only が発動してしまうので、少なくとも `(str == "true")` の方が良いと思います。
ただ、その上で、 str == "true", str == "false", otherwise の3通りに分けて、otherwiseではエラー出力しておしまい、という方が、適切な動作、ですかね。